hooks/post_gen_project.py

```diff
@@ -72,177 +52,87 @@ def print_thankyou():
     )
 
 
-def remove_vue3_files():
-    shutil.rmtree(os.path.join("clients", "vue3"))
+def get_random_secret_key():
+    # Vendoring Django's get_random_secret_key() so users don't need to install it
+    chars = "abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)"
+    return "".join(secrets.choice(chars) for i in range(50))
```

Comment:
Instead of the complicated `generate_random_string()` function, this is a copy/paste of what Django does to generate secret keys. I did try importing Django, but that felt like a heavy dependency to require for someone just trying to generate the code first.